MapleServer2/Tools/GameCommandActions.cs

```diff
@@ -20,12 +20,28 @@ public static void Process(GameSession session, string command)
             string[] args = command.ToLower().Split(" ", 2);
             switch (args[0])
             {
+                case "fi":
+                    session.SendNotice($"map: {session.Player.MapId} instance:{session.Player.InstanceId}");
+                    break;
                 case "commands":
                     ProcessCommandList(session);
                     break;
                 case "completequest":
                     ProcessQuestCommand(session, args.Length > 1 ? args[1] : "");
                     break;
+                case "oneshot":
+                    Player player = session.Player;
+                    if (player.GmFlags.Contains("oneshot"))
+                    {
+                        player.GmFlags.Remove("oneshot");
+                        session.SendNotice("oneshot mode disabled");
+                    }
+                    else
+                    {
+                        session.Player.GmFlags.Add("oneshot");
+                        session.SendNotice("oneshot mode enabled");
+                    }
+                    break;
```

Comment:
Same here

Reply:
This one I would leave in anyway. I think such a command is indispensable for debugging combat.